build.gradle

```diff
@@ -182,22 +182,24 @@ dependencies {
 
     implementation "io.github.jhipster:jhipster-framework"
     implementation "org.springframework.boot:spring-boot-starter-cache:${spring_boot_version}"
-    implementation "io.micrometer:micrometer-registry-prometheus:1.5.1"
+    implementation "io.micrometer:micrometer-registry-prometheus:1.3.5"
```

Comment:
why do we need an earlier version?

Reply:
I cannot remember what was the initial cause, but I've just tested it with 1.5.1 and it seems to work, so I've reverted this change.